sdk/eventhub/Azure.Messaging.EventHubs/samples/Program.cs

```diff
@@ -43,7 +53,8 @@ public static class Program
         ///
         private static readonly string[] ExtraOptionsForSamples = new[]
         {
-            "Explore Identity Samples"
+            "Explore Identity Samples",
```

Comment:
If we don't have an actionable plan for updates to the Event Hubs sample template, contributing guide, and (possibly) read me to allow using the Schema Registry, I'd prefer that we don't have the runner list out the samples for now.   Leaving the sample file and this infrastructure is a good idea, though.

Reply:
I've hidden the SchemaRegistry sample submenu. I've created an issue for tracking re-enabling it and updating the related documentation. https://github.com/Azure/azure-sdk-for-net/issues/15463